src/attestation_types/quote/sigdata.rs

```diff
@@ -31,6 +32,26 @@ impl From<&[u8; 64]> for ECDSAP256Sig {
     }
 }
 
+impl ECDSAP256Sig {
+    /// Returns r component
+    pub fn r(&self) -> &[u8; 32] {
+        &self.r
+    }
+
+    /// Returns s component
+    pub fn s(&self) -> &[u8; 32] {
+        &self.s
+    }
```

Comment:
`r` and `s` are already `pub` ... why these methods?

Reply:
Good point, I removed all of these.